docker-compose.yml

```
@@ -0,0 +1,143 @@
+      - ssgdeal-network
+
+  user-service:
+    build:
+      context: ./user_service
+      dockerfile: Dockerfile
+    container_name: user-service
+    ports:
+      - "8081:8081"
+    networks:
+      - ssgdeal-network
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)